app/src/main/java/com/github/se/polyfit/ui/flow/RecipeRecFlow.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.se.polyfit.ui.flow
+
+import androidx.compose.runtime.Composable
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.github.se.polyfit.ui.navigation.Navigation
+import com.github.se.polyfit.ui.navigation.Route
+import com.github.se.polyfit.ui.screen.recipeRec.RecipeRecommendationMoreDetailScreen
+import com.github.se.polyfit.ui.screen.recipeRec.RecipeRecommendationScreen
+import com.github.se.polyfit.viewmodel.recipe.RecipeRecommendationViewModel
+
+@Composable
+fun RecipeRecFlow(
+    recipeRecommendationViewModel: RecipeRecommendationViewModel = hiltViewModel(),
+) {
+  val navigation = rememberNavController()
```

Comment:
###Question
- Is there any specific reason for passing the `NavController` directly to the `RecipeRecommendationScreen`?

Reply:
The way the generic screen is implemented, it needs a `NavController`. It is not within the scope of this PR to modify that component. Feel free to create a new task on the scrum board if you think it is necessary or would be a good idea to refactor that component.